src/scss/blocks/_status-list.scss

```diff
@@ -0,0 +1,58 @@
+/// PATTERN LIBRARY LOCATION
+/// https://web.dev/design-system/pattern/status-list
+.status-list {
+  list-style: none;
+  padding: 0;
+
+  li {
+    position: relative;
+    padding-inline-start: 25px;
+  }
+
+  li + li {
+    margin-block-start: 0.5em;
+  }
+
+  a {
+    color: currentColor;
+  }
+
+  /// The cross is the default icon
```

Comment:
This is a very minor nit, but what do you think about making the checkbox the default icon? I feel like I've tended to see checkmark lists more often on the site but that is purely a half remembered anecdote. If in your research you saw folks mostly using 'x' then feel free to ignore.

Reply:
Yeh I did cross as default because it's less code, but can deffo flip that